resources/services/kubernetes/clusters.go

```diff
@@ -0,0 +1,1185 @@
+								Type:        schema.TypeString,
+							},
+							{
+								Name:        "code",
+								Description: "\"UNKNOWN\" - UNKNOWN indicates a generic condition   \"GCE_STOCKOUT\" - GCE_STOCKOUT indicates that Google Compute Engine resources are temporarily unavailable   \"GKE_SERVICE_ACCOUNT_DELETED\" - GKE_SERVICE_ACCOUNT_DELETED indicates that the user deleted their robot service account   \"GCE_QUOTA_EXCEEDED\" - Google Compute Engine quota was exceeded   \"SET_BY_OPERATOR\" - Cluster state was manually changed by an SRE due to a system logic error   \"CLOUD_KMS_KEY_ERROR\" - Unable to perform an encrypt operation against the CloudKMS key used for etcd level encryption   \"CA_EXPIRING\" - Cluster CA is expiring soon",
+								Type:        schema.TypeString,
+							},
+							{
+								Name:        "message",
+								Description: "Human-friendly representation of the condition",
+								Type:        schema.TypeString,
+							},
+						},
+					},
+					{
+						Name:        "gcp_kubernetes_cluster_node_pool_config_taints",
+						Description: "key, value, and effect",
+						Resolver:    fetchKubernetesClusterNodePoolConfigTaints,
+						Columns: []schema.Column{
+							{
+								Name:        "cluster_node_pool_cq_id",
+								Description: "Unique CloudQuery ID of gcp_kubernetes_cluster_node_pools table (FK)",
+								Type:        schema.TypeUUID,
+								Resolver:    schema.ParentIdResolver,
+							},
+							{
+								Name:        "effect",
+								Description: "\"EFFECT_UNSPECIFIED\" - Not set   \"NO_SCHEDULE\" - NoSchedule   \"PREFER_NO_SCHEDULE\" - PreferNoSchedule   \"NO_EXECUTE\" - NoExecute",
+								Type:        schema.TypeString,
+							},
+							{
+								Name:        "key",
+								Description: "Key for taint",
+								Type:        schema.TypeString,
+							},
+							{
+								Name:        "value",
+								Description: "Value for taint",
+								Type:        schema.TypeString,
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+
+func fetchKubernetesClusters(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+	call := c.Services.Container.Projects.Locations.Clusters.List("projects/" + c.ProjectId + "/locations/-")
+	list, err := c.RetryingDo(ctx, call)
+	if err != nil {
+		return err
+	}
+	output := list.(*container.ListClustersResponse)
+
+	res <- output.Clusters
+	return nil
+}
+
+func castParentToCluster(parent *schema.Resource) (*container.Cluster, error) {
+	p, ok := parent.Item.(*container.Cluster)
+	if !ok {
+		return nil, fmt.Errorf("expected *container.Cluster but got %T", p)
+	}
```

Comment:
I don't think we are supposed to be checking that the cast was successful anymore